tests/components/mysensors/conftest.py

```diff
@@ -232,6 +246,34 @@ def power_sensor(gateway_nodes: dict[int, Sensor], power_sensor_state: dict) ->
     return node
 
 
+@pytest.fixture(name="rgb_node_state", scope="session")
+def rgb_node_state_fixture() -> dict:
+    """Load the rgb node state."""
+    return load_nodes_state("mysensors/rgb_node_state.json")
```

Comment:
For a future PR, you could consider removing the domain from the path and using `load_fixture(fixture_path, DOMAIN)` instead.

Reply:
Thanks! I'll add it to the list. 👍

Reply:
https://github.com/home-assistant/core/pull/84618